core/dotnet2.1/build-run.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Useful for local testing.
+# USE WITH CAUTION !!
+
+# Removes all previously built instances.
+
+docker build -t openwhisk-runtime-dotnet .
+
+echo ""
+echo "  ---- RUNNING ---- "
+echo ""
+
+docker run --rm -it -p 8080:8080 openwhisk-runtime-dotnet
```

Comment:
you don't need this in that the gradle command will rebuild the container for you.